contrib/experimental/great_expectations_experimental/metrics/column_values_string_integers_monotonically_increasing.py

```diff
@@ -0,0 +1,153 @@
+from typing import Optional
+
+import numpy as np
+
+from great_expectations.core import ExpectationConfiguration
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+)
+from great_expectations.execution_engine.execution_engine import (
+    MetricDomainTypes,
+    MetricPartialFunctionTypes,
+)
+from great_expectations.execution_engine.sqlalchemy_execution_engine import (
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.metrics.import_manager import (  # sa,
+    F,
+    Window,
+    sparktypes,
+)
+from great_expectations.expectations.metrics.map_metric_provider import (
+    ColumnMapMetricProvider,
+    column_function_partial,
+    metric_partial,
+)
+from great_expectations.validator.metric_configuration import MetricConfiguration
+
+
+class ColumnValuesStringIntegersMonotonicallyIncreasing(ColumnMapMetricProvider):
+    function_metric_name = "column_values.string_integers.monotonically_increasing"
+
+    @column_function_partial(engine=PandasExecutionEngine)
+    def _pandas(self, data, _metrics, **kwargs):
```

Comment:
The column being passed should be referred to as `column` instead of `data` like in `column_values_increasing.py`.